### PR TITLE
Added equivalent axiom to 'acute leukemia'

### LINKED ISSUE #3231: add axiom to acute leukemia
add Equivalent To:

'leukemia (disease)'
 and ('has modifier' some acute)

### PR DESCRIPTION
**Pull request by sabrinatoro**
Fix #3231

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@sabrinatoro this failed because of the OMIM subsumption rule:
image

This means an OMIM class cannot be a subclass of another OMIM class, because all OMIM classes have a mutation in a specific gene, so a class with a mutation in gene A cannot be a child of a class with a mutation in gene B, because then that would mean the child would have a mutation in gene B too.

But in this case, I don't see this as the case (if you look at the two parents, 'leukemia (disease)' and 'acute disease', neither of these have an xref to OMIM.

**Comment by nicolevasilevsky** [comment]
I triggered the build again, we'll see if it fails again.

To do this:
on the command line:
navigate to mondo/src/ontology
`git checkout issue-3231`
`git commit --allow-empty -m "trigger build"`
`git push`

**Comment by nicolevasilevsky** [comment]
@matentzn I don't know why this is failing, am I missing something?
Thanks for your help!

**Comment by matentzn** [comment]
acute myeloid leukemia (mapped to OMIM:601626) is a subclass of "acute leukemia (disease)" which is mapped to OMIM:308960 - don't forget to run the reasoner before checking for the root cause of this error!

**Comment by nicolevasilevsky** [comment]
Thanks @matentzn! I missed that acute leukemia (disease) had an OMIM xref.

@sabrinatoro and I will review this together and fix it tomorrow.

**Comment by sabrinatoro** [comment]
the issue is that "acute leukemia" has 
Mondo:equivalentTo OMIM:308960
(and the children terms have different OMIM-equivalent)

OMIM:308960 refers to "leukemia, acute, X-linked", which is not equivalent to the general "acute leukemia" --> create new term for this OMIM-id

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -437695,6 +437701,15 @@ intersection_of: has_modifier MONDO:0700005 ! idiopathic
 property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/3169 xsd:string
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700060
+name: leukemia, acute, X-linked
+xref: OMIM:308960 {source="MONDO:equivalentTo"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:437707]: can you please add a text def, following this pattern:
https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/x_linked.yaml

The def would be:
X-linked form of acute leukemia.
dbxref: MONDO:patterns/x_linked

**Review by nicolevasilevsky — COMMENTED** [review]
see comment above ⬆️

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**